scoresheets/Receptionist.tex

```diff
@@ -1,16 +1,15 @@
-The maximum time for this test is \textbf{5 minutes}.
+The maximum time for this test is 5 minutes.
 
-\begin{scorelist}
+\begin{scorelist}[startbutton=false]
 	\scoreheading{Main Goal}
-	\scoreitem{100}{Introduce the 1st guest to John}
-	\scoreitem{150}{Offer an empty seat to the 1st guest}
-	\scoreitem{100}{Introduce the 2nd guest to John and 1st guest}
-	\scoreitem{150}{Offer an empty seat to the 2nd guest}
-	\penaltyitem[2]{50}{Misunderstanding the name of a guest}
+	\scoreitem[2]{250}{Introduce a new guest to every other guest and offer a seat}
```

Comment:
Is there a penalty for offering an occupied seat, or does that lose all 250pts?

Reply:
Loose all points imo. The robot can just check with the guest if the seat it offered is actually free and look for a different one otherwise.

Reply:
Okay so as long as the person gets seated in the end, there is no penalty; if the person never gets an empty seat, no points. Maybe change "offer a seat" to "get the guest seated" to be clear?

Reply:
Yeah, this is somewhat difficult to decide for each test. I think here, I'm fine with it because the person is naturally there and it is reasonable for the robot to ask whether the person wants to sit there. The reward for a team that gets it right on the first try is that they have more time.